spec/controllers/application_controller_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
-# rubocop:disable RSpec/EmptyExampleGroup
 RSpec.describe ApplicationController do
+  describe 'inheritance' do
+    subject { described_class }
+
+    it { should be < ActionController::API }
```

Comment:
Depcrecated syntax, please use as in example below:
```ruby
it { expect(described_class).to be < ActionController::API }
```

Also good practice to name your subjects.